tables/__init__.py

```diff
@@ -0,0 +1,6 @@
+import tables.checks  # noqa - adds custom checks to panderas built in set of checks
```

Comment:
I originally thought it could be best to keep it separate to try and protect against it getting tangled up in application specific code / dependencies and to just keep the calling code and schemas stored in the application code.. It's not exactly a full proof method of doing this though and does go against current convention on the repo